### PR TITLE
[irods/irods#7265] post-4.3.1 CMake touch-ups

### PR DESCRIPTION
**Pull request by SwooshyCueb**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,19 +8,15 @@ set(IRODS_PACKAGE_REVISION "1")
...
-set(CMAKE_SKIP_BUILD_RPATH OFF)
-set(CMAKE_SKIP_INSTALL_RPATH OFF)
-set(CMAKE_SKIP_RPATH OFF)
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH OFF)
-set(CMAKE_BUILD_RPATH_USE_ORIGIN ON)
+# Include this module to inherit iRODS's defaults for RPATH/RUNPATH handling
+include(IrodsRunpathDefaults)
 
 include(${IRODS_TARGETS_PATH})
```
Comment (alanking) [inline CMakeLists.txt]:
```suggestion
include("${IRODS_TARGETS_PATH}")
```

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by SwooshyCueb** [comment]
\#'d

**Review by alanking — COMMENTED** [review]
Had one suggestion - not sure how important it is. :) Sorry, was too slow...

File: CMakeLists.txt (hunk 1 above)
Comment (SwooshyCueb) [reply CMakeLists.txt]: Nice catch

**Comment by SwooshyCueb** [comment]
Ready for merge

**Review by alanking — APPROVED** [review]

**Pull request merged**